client/src/components/screens/PatientSubmission/ClinicalInformation.jsx

```diff
@@ -236,7 +236,18 @@ class ClinicalInformation extends React.Component {
         </div>
         <div className="phenotypeSecondLine" key={`input-${hpoIndex}`}>
           <Form.Item>
-            <Input placeholder="Ajouter une note…" value={defaultValue()} size="small" onChange={event => this.handleHpoNoteChanged(event.target.value, hpoIndex)} className="input hpoNote" />
+            {getFieldDecorator(`hpoNotes[${hpoIndex}]`, {
```

Comment:
We don't need the field decorator here, `hpoNotes` is not used anymore. More precisely, it's used in `createHPOResourceList` which should also be removed (also not used)